db/blob/db_blob_compaction_test.cc

```diff
@@ -390,6 +390,34 @@ TEST_F(DBBlobCompactionTest, CompactionFilterReadBlobAndKeep) {
   Close();
 }
 
+TEST_F(DBBlobCompactionTest, MergeBlobWithBase) {
+  Options options = GetDefaultOptions();
+  options.create_if_missing = true;
+  options.enable_blob_files = true;
+  options.min_blob_size = 0;
+  options.merge_operator = MergeOperators::CreateStringAppendOperator();
+  options.disable_auto_compactions = true;
+
+  DestroyAndReopen(options);
```

Comment:
Minor but I think you could simply call `Reopen(options)` here and remove `options.create_if_missing = true;` above.